### ISSUE TITLE
Grand theft Palomino bugged...

### ISSUE THREAD

**Issue by glerpdog1973** [dalaranwow/dalaran-wow#5290]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behavior**: Tell us what happens in detail (Where, When, What, Why, Who)

**Expected behavior**: Tell us what should happen instead

**Steps to reproduce the problem**:

Just started my death knight quest line and tried to do the horse quest but it seems to be broken, The quest is called "Grand theft Palomino" I have went and retrieved a horse 5 separate times and Salanar the Horseman does not recognize I have captured a horse. Please let me know if the issue is something that is a known issue or something brand new. Thank you for your time.

Here is a link to a video in which the quest is functioning properly.

https://www.youtube.com/watch?v=ghGomNcfMxQ

**Include proofs for this behavior**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**
